docs/release-notes/index.rst

```diff
@@ -0,0 +1,253 @@
+.. _release-notes:
+
+Release notes
+=============
+
+This page lists past release notes for Starcraft, summarising new features, bug
+fixes and backwards-incompatible changes in each version. It also contains the
+release and support policies for Starcraft.
+
+
+Current releases
+----------------
+
+
+<latest release>
+~~~~~~~~~~~~~~~~
+
+- <link to latest release, update, such as 8.2.0>
+- <link to latest release, update, such as 8.1.0>
+- <link to latest release, initial, such as 8.0.0>
+
+
+<parallel release>
+~~~~~~~~~~~~~~~~~~
+
+<If necessary, add guidance and caveats about these older releases, such as
+"Snapcraft 7 is available for building core18 snaps. When building for a newer
+base, use Snapcraft 8.">
+
+- <link to parallel release, update, such as 7.2.0>
+- <link to parallel release, update, such as 7.1.0>
+- <link to parallel release, initial, such as 7.0.0>
+
+
+Past releases
+-------------
+
+
+<past release>
+~~~~~~~~~~~~~~
+
+- <link to past release, update, such as 6.2.0>
+- <link to past release, update, such as 6.1.0>
+- <link to past release, initial, such as 6.0.0>
+
+
+.. _release_policy_and_schedule:
+
+Release policy and schedule
+---------------------------
+
+Canonical is committed to supporting the <"latest major release" or "last two
+major releases"> of Starcraft. <Optional: "We forward-port changes in older
+releases to the latest release, if they're compatible.">
+
+Starcraft is released according to the Semantic Versioning 2.0.0 scheme with
```

Comment:
This is true for the libraries but not for the applications
I suppose for the applications we could add a separate policy later

Reply:
What's the difference?  I think this aligns with Snapcraft

Reply:
For the applications, in theory the only time where we're allowed to do breaking changes is when updating the project's `base`. That is, a `core22` project should build the same in Snapcraft 7 and Snapcraft 8, but that same project could be built in a different, incompatible manner if updated to base `core24`

Reply:
So in a sense the `base` is the hidden "supermajor" version

Reply:
This differs with what I discussed with Claudio. I had the understanding that a breaking change triggers a major release.

Reply:
Here's an example: https://forum.snapcraft.io/t/migrate-to-core24/39393/6

Reply:
@tigarmo How about this? bdf786dd13afdde14e0c779412377bb4487e7e6d

Reply:
looks good to me!